discirc/irc_bot.py

```diff
@@ -70,6 +72,8 @@ def on_connect(self):
         self.send('USER', user=self.nick, realname='Discord gateway')
         if self.password:
             self.send('PASS', password=self.password)
+
+    def on_motddone(self, message):
```

Comment:
Could you just add a small docstring to the function ?
People may not understand what `motdone` is.